ocfweb/docs/docs/staff/procedures/accounts/renaming.md

```diff
@@ -0,0 +1,100 @@
+[[!meta title="Renaming Accounts"]]
+
+Sometimes OCF accounts need to be renamed. Users sometimes make mistakes
+when creating their accounts, or group accounts might need to be changed,
+or for several other reasons.
+
+Renaming accounts touches several parts of the OCF's infrastructure, and
+as parts are changed, this document may go out of date, but renaming an
+account is an informative way to learn about how some parts of the OCF work.
+
+The steps to rename an account are roughly the following:
+1. Move the homedir and webdir
+2. Rename the user in Kerberos
+3. Rename the user in LDAP
+4. Change the user's symlinks
+5. Change the user's name in the print quota system
+6. Update the user's crontabs
+7. Update the user's MySQL database if they have one
+8. Document the change using `note`
+
+## Performing the rename
+
+Do this on supernova. You'll need root and an /admin principal.
+
+### Move user directories
+
+    $ mv /home/<old[0]>/<old[:1]>/<old> /home/<new[0]>/<new[:1]>/<new>
+    $ mv /services/http/user/<old[0]>/<old> /services/http/user/<new[0]>/<new>
```

Comment:
This is pretty hard to read. I suggest you make up an old and new name as an example, such as `oldname` and `newname`. Then the first line would look like
```
$ mv /home/o/ol/oldname /home/n/ne/newname
```